README.md

```diff
@@ -15,20 +15,20 @@ Supported releases and dependencies are shown below.
 
 | kamon  | status | jdk  | scala            
 |:------:|:------:|:----:|------------------
-|  1.0.0 | stable | 1.8+ | 2.10, 2.11, 2.12
+|  1.0.4 | stable | 1.8+ | 2.10, 2.11, 2.12
```

Comment:
And, we just published 1.0.5, please update to that number :slightly_smiling_face: